lock-api/src/main/java/com/palantir/lock/AtlasCellLockDescriptor.java

```diff
@@ -20,23 +20,26 @@
 import com.google.common.base.Preconditions;
 import com.google.common.base.Strings;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
 /**
  * A descriptor for a {@link ReadWriteLock}, identified by a lock ID (a unique
  * string).
  *
  * @author jtamer
  */
-public class AtlasCellLockDescriptor {
+public final class AtlasCellLockDescriptor {
 
     private AtlasCellLockDescriptor() {
         // cannot instantiate
     }
 
     /** Returns a {@code LockDescriptor} instance for the given table, row, and column. */
+    @SuppressFBWarnings("DM_DEFAULT_ENCODING") // TODO (jkong): Investigate whether we can fix an encoding
```

Comment:
Actually, because Glenn migrated us from FindBugs to ErrorProne this isn't needed anymore. Yep, agree that fixing an encoding is potentially breaking which is why I decided to suppress the warning rather than enforce UTF-8 or anything else.